packages/@expo/fingerprint/src/utils/__tests__/Profile-test.ts

```diff
@@ -6,6 +6,6 @@ it(`respects input types`, () => {
     if (typeof b === 'string') return -1;
     return 0;
   });
-  expect(profile(fn)('a', 'b')).toBe(1);
+  expect(profile({} as any, fn)('a', 'b')).toBe(1);
```

Comment:
```suggestion
  expect(profile({} as NormalizedOptions, fn)('a', 'b')).toBe(1);
```

nit: explicit type
also need to have `import type { NormalizedOptions } from '../../../Fingerprint.types';` at top